packages/utilities/moment-timezone-include/src/index.js

```diff
@@ -0,0 +1,19 @@
+import webpack from 'webpack';
+import writeNewTimezoneData from './writeNewTimezoneData';
+import replaceOriginalTimezoneData from './replaceOriginalTimezoneData';
+
+function MomentTimezoneInclude(options = {}) {
+  const startYear = options.startYear || -Infinity;
+  const endYear = options.endYear || Infinity;
+
+  writeNewTimezoneData(startYear, endYear);
+
+  return new webpack.NormalModuleReplacementPlugin(
+    /data[\\/]packed[\\/]latest\.json$/,
+    resource => {
+      resource.request = replaceOriginalTimezoneData(); // eslint-disable-line no-param-reassign
```

Comment:
To prevent the `eslint-disable-line`, could you not do the following:

```suggestion
      ...resource,
      request: replaceOriginalTimezoneData()
```

Reply:
Ah, we cant do this as it doesnt except a new object back, but a mutation of the arguement